src/server/system_services/stats_aggregator.js

```diff
@@ -684,15 +685,17 @@ async function object_usage_scrubber(req) {
 
 //_.noop(send_stats_payload); // lint unused bypass
 
-function send_stats_payload(payload) {
+async function send_stats_payload(payload) {
     var system = system_store.data.systems[0];
+    const zipfile = await zip_utils.zip_from_files([{ data: JSON.stringify(payload), path: 'payload.txt' }]);
```

Comment:
Actually, I see that we use request() with `gzip: true` so it should already do compression on the wire?

Reply:
As far as I know, the message is already compressed. I don't get this change.

Reply:
I guess you guys are correct. I found this in the phone home code: ```app.use(bodyParser.json({
    **inflate: true,**
    limit: 1024 * 1024,
    type: req => (req.headers['content-type'].indexOf('application/json') !== -1 ||
        req.headers['content-type'].indexOf('application/gzip') !== -1)
}));
```
which does the uncompress,. @guymguym @nimrod-becker  So, the only fix is just to change the limit to something bigger, right?
to avoid this: "Error: request entity too large"
the size of the string was 1.6MB(with 10K buckets) so I guess it shouldn't get much bigger than now - maybe 5...

Reply:
The body limit is already 1MB which is quite big.
We probably don't want to grow the message so much as this runs all the time.
The other options are to to shrink/remove the buckets information from this request.
Can you shrink the info per bucket? 
I suggest to truncate the list of buckets sent to 1000 but first sort it by recent activity.
For all the rest we can send an aggregated stats.

Reply:
will truncate 1000 buckets sorted by number of objects